client/components/RelatedProducts/Related/RelatedProductCard.jsx

```diff
@@ -14,7 +14,8 @@ import Rating from '@material-ui/lab/Rating';
 const useStyles = makeStyles({
   root: {
     minWidth: 250,
-    minHeight: 300,
+    minHeight: 350,
+    maxHeight: 350,
```

Comment:
It's so weird that material-ui makes you set a max & min height to set the size of something. I did something similar in one place, but it feels strange.